charts/runner-manager/values-runner-manager-gcloud.yaml

```diff
@@ -0,0 +1,57 @@
+gcloudServiceAccount: "vault:secret/data/runner-manager#GCLOUD_SERVICEACCOUNT"
+githubToken: "vault:secret/data/runner-manager#GITHUB_TOKEN"
+redhatUsername: "vault:secret/data/runner-manager#REDHAT_USERNAME"
+redhatPassword: "vault:secret/data/runner-manager#REDHAT_PASSWORD"
+#redis:
+#  password: "vault:secret/data/runner-manager#rREDIS_PASSWORD"
+#allowedSshKeys: "vault:secret/data/runner-manager#allowed_ssh_keys"
+githubOrganization: scalanga-devl
+deployment:
+  ingress:
+    #enabled: true
```

Comment:
Why did you comment on the whole igress? If it's done on purpose to test the service without and then implement it, I'm fine with it

Reply:
yes exactly to test the service